libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -267,10 +233,15 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
  */
 - (NSArray<SFUserAccount*> *)accountsForInstanceURL:(NSURL *)instanceURL;
 
-/** Adds a user account
- @param acct The account to be added
+/** Adds/Updates a user account
+ @param userAccount The account to be added
+ */
+- (BOOL)saveAccountForUser:(SFUserAccount *)userAccount error:(NSError **) error;
```

Comment:
@khawkins  I don't see the harm in being able to provide CRUD operations for Accounts. Provides a way to manage accounts if needed. Same reason why all the other Account Management functions are available through this interface.